__tests__/_helper.ts

```diff
@@ -178,3 +178,14 @@ export function returnResponseText(body: string): RestResolver {
 export function returnResponseJson(body: any): RestResolver {
   return (_req, res, ctx) => res.once(ctx.json(body))
 }
+
+export function returnResponseApi(data: unknown) {
+  return returnResponseJson({ data })
+}
+
+/*
+export function returnResponseText(body: string, args : { status? : number }|undefined): RestResolver {
+  const status = args?.status ?? 200
+  return (_req, res, ctx) => res.once(ctx.body(body))|
+ }
+ */
```

Comment:
Why is this function comment out?!

Reply:
I tried to implement the status argument in the function unsuccessfully. That's why i left it commented out for now